internal/provider/harness_base_resource.go

```diff
@@ -232,7 +232,7 @@ func (r *BaseHarnessResource) do(ctx context.Context, req framework.CreateOrUpda
 	// At this point (update/create), we should have an inventory Set during ModifyPlan()
 	inv, ok := r.store.inv.Get(data.Inventory.Seed.ValueString())
 	if !ok {
-		return []diag.Diagnostic{diag.NewErrorDiagnostic("failed to get inventory", err.Error())}
+		return []diag.Diagnostic{diag.NewErrorDiagnostic("failed to get inventory", fmt.Sprintf("inventory %s", data.Inventory.Seed.ValueString()))}
```

Comment:
thoughts on adding inventory seed to the main message so that the error can remain the detail, or otherwise incorporating the error message into the detail? tangentially, it did occur to me we might want to standardize on some convention since some summaries include resource ID and some dont. the ones that don't get merged by TF due to being similar which is interesting but I think altogether not useful because it erases the full list of everything that was reported

Reply:
there is no error message here. golint was throwing a warning at me I missed in a previous PR.

the `err.Error()` being checked here is accidentally using the `err` variable from the earlier `r.store.Logger()` call

Reply:
OOPS, jeez